Music.py

```diff
@@ -133,11 +133,24 @@ def process_sequences(rom, sequences, target_sequences, disabled_source_sequence
         if cosmetic_name not in disabled_target_sequences:
             target_sequences.append(target)
 
+    # If present, load the file containing custom music to exclude
+    try:
+        with open(os.path.join(u'.', u'data', u'custom_music_exclusion.txt')) as excl_in:
```

Comment:
```suggestion
        with open(os.path.join(data_path(), u'custom_music_exclusion.txt')) as excl_in:
```
also need to import it `from Utils import data_path`

Reply:
Didn't know this was a thing, thanks for pointing it out!